doc/contributing/writing-tests.md

```diff
@@ -12,6 +12,8 @@ with code `0` on success. A test will fail if:
 * It never exits. In this case, the test runner will terminate the test because
   it sets a maximum time limit.
 
+it is recommended to [use `node:test`][] for handling these requirements
```

Comment:
```suggestion
It is recommended to [use `node:test`][] for handling these requirements.
```

Reply:
```suggestion
Use [`node:test`][] for handling these requirements.
```

(And update the reference at the end.)

Ref: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/recommend